### ISSUE TITLE
Use on Fedora (15are05)

### ISSUE THREAD

**Issue by winkelnp** [0xless/battmngr#9]
Worked fine on Manjaro, recently moved to Fedora 36 and now the module acpi_call cannot be found. DNF cannot seem to find a package by that name.

Any help with getting this to run? I really appreciated it on manjaro, especially the battery conservation mode.

**Comment by 0xless** [comment]
Hi @winkelnp, thanks for showing interest in my little project. 
I've never tried using battmngr on Fedora, but I'll try to figure out how to install the dependencies.

**Comment by winkelnp** [comment]
Thanks :)

**Comment by 0xless** [comment]
Hi, @winkelnp, I couldn't properly test `battmngr` on fedora without installing it on my laptop.
I could find some info about acpi_calls on fedora here: https://linrunner.de/tlp/installation/fedora.html#thinkpads-only-external-kernel-modules

I think that `akmod-acpi_call` module will do! Let me know if it works.

**Comment by winkelnp** [comment]
Sorry for the late reply, but it looks like I can't use `akmod-acpi_call` without switching to `kernel-devel` which I do not want to do on my laptop... It also says something about not being needed for kernel 5.17+, and I'm on kernel 5.18.19, so I shouldn't need these anyway. it also mentions something about a "Thinkpad Extras" repo, that I cannot find anywhere...

I'll look again to see if I can find any other way of getting `acpi_call` on fedora. I'll update here if and when I find something

**Comment by leonardozilli** [comment]
Hi, did you ever find a solution to this?

**Comment by winkelnp** [comment]
No, I did not. All the acpi_call things I tried did not work or actually add to the kernel, if anyone figures something out please let me know
